### ISSUE TITLE
Screen Readers do not announce new playback rate after switching to new rate using keyboard only commands

### ISSUE THREAD

**Issue by feliperobledo** [videojs/video.js#7121]
## Description

When I have the playback rate option as outlined above, screen readers do not properly announce the change in playback rate after the rate changes.

You can test the above in this codepen: https://codepen.io/feliperobledo/pen/bGBjXXQ

## Steps to reproduce

1. Open the screen reader software of your choice
2. Open the codepen above
3. Tab or click on the playback rate button to switch the rate.

At this point the screen reader will simply announce, `"Playback rate"`, but it will not announce the current play speed.

### Expected

Consider that the playback rate button has options to go `1`, `1.2`, `1.5`, or `2`.
If the label on the playback rate button is `1x` and I click on the playback rate button, the screen reader software should announce `"Playback rate 1 point 2 x"`, meaning it announces the new label selected rate.

### Actual

The screen reader simply announce `"Playback rate"`

### Error output
N/A

### Proposed solution

<img width="429" alt="Screen Shot 2021-02-23 at 12 06 32 PM (1)" src="https://user-images.githubusercontent.com/6238302/110151012-ef262200-7dad-11eb-81f7-76b0cefd9efd.png">

That we add `id`s to the elements that hold the information we want announced, and we tell screen readers that the playback rate button's label is a combination of those elements, as shown above.

I've personally tried it on:
- macOS - Chrome - Screen Reader extension
- macOS - Chrome - VoiceOver

And it works.

#### videojs
`7.9.7`

#### browsers
All of which that have a playback rate button that is not a menu but just a button that changes rate

#### OSes
All

### plugins
Self hosted `videojs-offset`
`videojs-contrib-quality-levels`

**Comment by gkatsev** [comment]
@OwenEdwards would love to get your input on this. Also, given that there's no way to open the menu for keyboard/SR users, should we even be announcing it as a menu to SR users?

**Comment by gkatsev** [comment]
I think we likely want to change up the behavior of this to be in-line with other menus, particularly after reading https://css-tricks.com/in-praise-of-the-unambiguous-click-menu/. Meaning, clicking on it should open up the menu rather than toggle the playbackRate. I've added it to our [major tracking issue.](https://github.com/videojs/video.js/issues/7068)

**Comment by feliperobledo** [comment]
@gkatsev if we were to contribute a PR to add the proposed change, would you be opposed to it? This is knowing with full conscience it is throw-away code for the next release.

**Comment by gkatsev** [comment]
We definitely would value contribution. Though, I wonder if we want to just change the behavior (where it matches other menus), either with a flag in the current control, or create a new playback rate control that could be configured and by default use the old control.

**Comment by feliperobledo** [comment]
@gkatsev we are okay with waiting for the refactor on the playback rate button to open the menu.
Also, thanks for the article you linked to. It was insightful.

**Comment by gkatsev** [comment]
@feliperobledo thankfully, @OwenEdwards provided a great workaround, it doesn't solve everything but should much improve the behavior. It'll be out in an upcoming patch release.

**Comment by feliperobledo** [comment]
@gkatsev sorry for the delayed response.
That's awesome! Thank you!

**Comment by gkatsev** [comment]
Cool, maybe I didn't quite understand what you had meant back then :)

**Comment by feliperobledo** [comment]
I need to work on those communication skills 🤦

**Comment by gkatsev** [comment]
I'm sure it wasn't an issue on your end, but on mine :D
